src/utils/dom.js

```diff
@@ -92,9 +92,15 @@ const sweetHTML = `
      <div class="${swalClasses.icon} ${iconTypes.error}">
        <span class="swal2-x-mark"><span class="swal2-x-mark-line-left"></span><span class="swal2-x-mark-line-right"></span></span>
      </div>
-     <div class="${swalClasses.icon} ${iconTypes.question}">?</div>
-     <div class="${swalClasses.icon} ${iconTypes.warning}">!</div>
-     <div class="${swalClasses.icon} ${iconTypes.info}">i</div>
+     <div class="${swalClasses.icon} ${iconTypes.question}">
+       <span class="swal2-icon-text">?</span>
```

Comment:
Add `icon-text` to [classes.js](https://github.com/sweetalert2/sweetalert2/blob/master/src/utils/classes.js#L32)

and then

`class="swal2-icon-text"` -> `class="${swalClasses['icon-text']}"`